src/components/Home.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { coinDetail, fetchCoin } from '../redux/coin/home';
+
+const Coin = () => {
+  const coinState = useSelector((state) => state.cryptoCoin);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCoin());
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1 className="logo">
+        coinX
+      </h1>
+      <table>
+        <div className="crypto-coins">
+          {coinState.map((coin) => (
+
+            <tr key={uuidv4()} className={coin.baseId}>
```

Comment:
Kindly add a filter that can filter out the listed categories i.e by the category name as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/206861688-b82f1298-6ef5-4714-91e8-0d5aa03a88e6.png)

Reply:
- [ ] As requested by the previous reviewer, what he meant by asking "the list of categories that could be filtered out by some parameter, for example by the category name." was that you should add a search input field that will filter and only show the results based on the input that has been typed in the input form. I hope this is more clear now.

![image](https://user-images.githubusercontent.com/86318284/206866263-aa3c6b2f-ce87-4562-ab76-91aa3214255a.png)